test/lendOrder.js

```diff
@@ -0,0 +1,84 @@
+const Reverter = require('./helpers/reverter');
+const Asserts = require('./helpers/asserts');
+const LendOrder = artifacts.require('./LendOrder.sol');
+
+contract('LendOrder', function(accounts) {
+  const reverter = new Reverter(web3);
+  afterEach('revert', reverter.revert);
+
+  const asserts = Asserts(assert);
+  const OWNER = accounts[0];
+  let lendOrder;
+
+  before('setup', async () => {
+    const borrower = accounts[3];
+      lendOrder = await LendOrder.deployed();
+      await lendOrder.register('VF',{from: borrower});
+      await reverter.snapshot();
+  });
+
+  it('should allow to repay', async () => {
+    const borrower = accounts[3];
+    const value = 1000;
+    await lendOrder.lendMoney(value, {from: borrower});
+    await lendOrder.repayMoney(value, borrower, {from: OWNER});
+    await lendOrder.balanceByAddr(borrower);
+    asserts.equal(0);
+  });
+
+  it('should fail on overflow when borrowing', async () => {
+    const borrower = accounts[3];
+    const value = '0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff';
+    await lendOrder.lendMoney(value, {from: borrower});
+    await asserts.throws(lendOrder.lendMoney(1, {from: borrower}));
+  });
+
+  it('should emit Borrowed event on borrow',async () => {
+    const borrower = accounts[3];
+    const value = 1000;
+    let result = await lendOrder.lendMoney(value, {from: borrower});
+      assert.equal(result.logs.length, 1);
+      assert.equal(result.logs[0].event, 'NewOrder');
+      assert.equal(result.logs[0].args.id, 0);
+      assert.equal(result.logs[0].args._amount.valueOf(), value);
+    });
+
+  it('should allow to borrow', async () => {
+    const borrower = accounts[3];
+    const value = 1000;
+    await lendOrder.lendMoney(value, {from: borrower});
+    await lendOrder.balanceByAddr(borrower);
+    asserts.equal(value);
+  });
+
+  it('should allow to regist new user', async () => {
+    const borrower = accounts[4];
+    const newUser = 'testUser';
+    let result = await lendOrder.register(newUser,{from: borrower});
+    assert.equal(result.logs.length, 1);
+    assert.equal(result.logs[0].event, 'NewUser');
+    assert.equal(result.logs[0].args._userName, newUser);
+  });
+
+  it('should emit Repayed event on repay', async () => {
```

Comment:
The name of the test doesn't match with content

Reply:
change to == > should emit NewOrder event on repay